aries_vcx/src/protocols/issuance/mod.rs

```diff
@@ -2,25 +2,23 @@ use aries_vcx_core::ledger::base_ledger::AnoncredsLedgerRead;
 use std::sync::Arc;
 
 use crate::errors::error::prelude::*;
-use crate::global::settings;
-use crate::protocols::issuance::actions::CredentialIssuanceAction;
 
-pub mod actions;
 pub mod holder;
 pub mod issuer;
 
-pub fn verify_thread_id(thread_id: &str, message: &CredentialIssuanceAction) -> VcxResult<()> {
-    if !settings::indy_mocks_enabled() && !message.thread_id_matches(thread_id) {
-        return Err(AriesVcxError::from_msg(
-            AriesVcxErrorKind::InvalidJson,
-            format!(
-                "Cannot handle message {:?}: thread id does not match, expected {:?}",
-                message, thread_id
-            ),
-        ));
-    };
-    Ok(())
-}
+// todo: restore, how to call this on arbitrary Aries message?
```

Comment:
did you not resolve this TODO? it looks like you're using src/handler/utils::verify_thread_id now for this

Reply:
ah yeah, it's thread_id verification is indeed still in place, as it was before

Reply:
removing this commented fragment